sample-plugin/src/main/java/com/github/jackchen/plugin/sample/extension/SampleExtension.kt

```diff
@@ -0,0 +1,20 @@
+package com.github.jackchen.plugin.sample.extension
+
+import org.gradle.api.provider.Property
+
+/**
+ * Configuration object for [com.github.jackchen.plugin.sample.SamplePlugin].
+ *
+ * @author airsaid
+ */
+abstract class SampleExtension {
+
+  /**
+   * Whether to enable debug mode. default disabled.
+   */
+  abstract val enableDebug: Property<Boolean>
+
+  init {
+    enableDebug.convention(false)
```

Comment:
Is convention false? Is it better to enable it?

Reply:
Yes, because it is related to debugging information, it is turned off by default, and it feels better to turn it on as needed.